logfire-api/logfire_api/__init__.py

```diff
@@ -19,8 +20,11 @@
         def configure(*args, **kwargs): ...
 
         class LogfireSpan:
+            def __init__(self, *args, **kwargs):
+                self._readable_span = ReadableSpan(name='')
+
             def __getattr__(self, attr):
-                return MagicMock()  # pragma: no cover
+                return self._readable_span.attributes.get(attr) # pragma: no cover
```

Comment:
```suggestion
                return getattr(self._readable_span, attr, MagicMock())
```

You're confusing OTel attributes with Python attributes. And we still want to cover all cases.